sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -52,6 +52,7 @@ enum {
 
 struct sof_sdw_codec_info {
 	const int id;
```

Comment:
how about change id to p_id ? it is confusing.

Reply:
we can use 100 characters so let's use part_id and vendor_id. No need to use any encoding.

Reply:
> we can use 100 characters so let's use part_id and vendor_id. No need to use any encoding.

Fixed